package.json

```
@@ -19,11 +19,13 @@
     "@types/react": "^16.9.23",
     "@types/react-dom": "^16.9.5",
     "@types/styled-components": "^5.0.1",
+    "prettier": "1.19.1",
     "typescript": "^3.8.3"
   },
   "scripts": {
     "dev": "next dev",
     "build": "next build",
+    "prettier": "prettier --write 'pages/**/*.{ts,tsx}'",
```

There will be code for repositories, models, constants and so on.

```suggestion
    "format": "prettier --write '**/*.{ts,tsx}'",
```